.changes/next-release/bugfix-botocore-39521.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "``botocore``",
```

Comment:
Don't see a precedent for `botocore`, which should just be an internal detail for V2. That said, don't really have a better suggestion so I'm okay with it, can always adjust later. See one occurrence of `HTTP`